scripts/r2s.py

```diff
@@ -112,21 +116,31 @@ def step1():
     tally_num = config.getint('step1', 'tally_num')
     flux_tag = config.get('step1', 'flux_tag')
     decay_times = config.get('step2', 'decay_times').split(',')
+    geom = config.get('step1', 'geom')
+    reverse = config.getboolean('step1', 'reverse')
+    num_rays = config.getint('step1', 'num_rays')
+    grid = config.getboolean('step1', 'grid')
 
+    load(geom)
+    # get meshtal info from meshtal file
     if structured:
         meshtal = Meshtal(meshtal,
                         {tally_num: (flux_tag, flux_tag + '_err',
                                      flux_tag + '_total',
                                      flux_tag + '_err_total')},
                         meshes_have_mats=False)
-    geom = config.get('step1', 'geom')
-    reverse = config.getboolean('step1', 'reverse')
-    num_rays = config.getint('step1', 'num_rays')
-    grid = config.getboolean('step1', 'grid')
+        m = meshtal.tally[tally_num]
+
+    # create the cell_fracs array before irradiation_steup
+    if m.structured:
```

Comment:
If using an unstructured mesh, m isn't set yet so I think this will cause an error.  Is this if statement necessary?

Reply:
@zxkjack123 do your R2S tests include any unstructured mesh?

Reply:
addressed in #57 
but it's still not being tested properly